lib/daru/io/exporters/excelx.rb

```diff
@@ -0,0 +1,114 @@
+require 'daru/io/exporters/base'
+
+module Daru
+  module IO
+    module Exporters
+      # Excelx Exporter Class, that extends `to_excelx_string` and `write_excelx` methods to
+      # `Daru::DataFrame` instance variables
+      class Excelx < Base
+        Daru::DataFrame.register_io_module :to_excelx_string, self
+        Daru::DataFrame.register_io_module :write_excelx, self
```

Comment:
Apart from standalone xlsx exporter, doesn't it make sense for the excel exporter to redirect here incase of `.xlsx` filename in excel exporter's write method? Though, for such a redirect to be possible, they would need to have similar kind of arguments. That is, formatting has to be supported by xlsx exporter like excel exporter.

Reply:
Not 100% sure. Yep, it looks humane, but if/when you'll add formatting support for Excel exporters -- conventions on formatting are pretty different in two versions of Excel files and corresponding libraries.
In fact, I believe that in **2017** support for Excel **1999-2003** format (xls) should be done in "maintenance" mode, while xlsx exporter should be shiny and friendly.